canonical-websites/build.snapcraft.io
# Add run script

## Done

Add run script to the project

## QA

- Check out this feature branch
- Run the site using the command ```./run```
- View the site locally in your web browser at: [http://0.0.0.0:8083/](http://0.0.0.0:8083/)
- Try to connect and navigate through the website
